core/lib/dal/src/blocks_dal.rs

```diff
@@ -1021,6 +1021,8 @@ impl BlocksDal<'_, '_> {
                 AND protocol_versions.bootloader_code_hash = $1 AND protocol_versions.default_account_code_hash = $2 \
                 AND commitment IS NOT NULL \
                 AND (protocol_versions.id = $3 OR protocol_versions.upgrade_tx_hash IS NULL) \
+                AND events_queue_commitment IS NOT NULL \
+                AND bootloader_initial_content_commitment IS NOT NULL
```

Comment:
This won't work because we don't calculate these commitments for pre-boojum batches. You should require them to be not null only if `!protocol_version_id.is_pre_boojum()`